conf/test.config

```diff
@@ -34,3 +34,7 @@ params {
     // Impute tools
     tools  = "glimpse1"
 }
+
+withName: 'NFCORE_PHASEIMPUTE:PHASEIMPUTE:VCF_CHUNK_GLIMPSE:GLIMPSE_CHUNK' {
+    ext.args   = ["--window-size 10000", "--window-count 400", "--buffer-size 5000", "--buffer-count 30"].join(' ')
+}
```

Comment:
Why does it have a different config than the default config at `imputation_glimpse1.config`?

Reply:
This is done to force multiple chunks in the test configuration, to verify that the chunks are correctly propagate.